.github/workflows/clone_with_pinned_version.yml

```
@@ -0,0 +1,49 @@
+name: JEDI Bundle Pinned Version Clone Test
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+
+jobs:
+  buildbundle:
+    name: Clone oops bundle
+    runs-on: ubuntu-latest
+
+    steps:
+
+    # Setup Python
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+
+    # Update conda
+    - name: Update conda
+      run: conda update -n base -c defaults conda
+
+    # Install pip
+    - name: Install pip
+      run: conda install pip
+
+    # Clone the code repo
+    - name: Clone code repo
+      uses: actions/checkout@v2
+      with:
+        lfs: true
+
+    # Install package
+    - name: Upgrade pip
+      run: $CONDA/bin/pip3 install --upgrade pip
+    - name: Install JEDI bundle and dependencies
+      run: $CONDA/bin/pip3 install --use-deprecated=legacy-resolver -r requirements.txt --user .
```

Is there a particular reason we're forcing `conda` here? If not, we're wasting CPU cycles on these install steps. Based on the [GitHub Actions docs for Python](https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python), I recommend the following:

```suggestion
    # Clone the code repo
    - name: Clone code repo
      uses: actions/checkout@v2
      with:
        lfs: true

    # Install package
    - name: Install JEDI bundle and dependencies
      run: python -m pip install --use-deprecated=legacy-resolver -r requirements.txt --user .
```

---

Yeah to be honest, the github action was a copy and paste of the "clone_a_jedi_bundle" one so I don't have any particular reasoning for the set up. I will try your suggestions